modules/mirror-registry-localhost-update.adoc

```diff
@@ -33,10 +38,16 @@ $ sudo ./mirror-registry upgrade -v
 * Users who upgrade _mirror registry for Red Hat OpenShift_ with the `./mirror-registry upgrade -v` flag must include the same credentials used when creating their mirror registry. For example, if you installed the _mirror registry for Red Hat OpenShift_ with `--quayHostname <host_example_com>` and `--quayRoot <example_directory_name>`, you must include that string to properly upgrade the mirror registry.
 ====
 
-* If you are upgrading the _mirror registry for Red Hat OpenShift_ from 1.2.z -> 1.3.0 and you used a specified directory in your 1.2.z deployment, you must pass in the new `--pgStorage` and `--quayStorage` flags. For example:
+* If you are upgrading _the mirror registry for Red Hat OpenShift_ from 1.3 -> 2.y and you used a custom quay configuration and storage directory in your 1.y deployment, you must pass in the `--quayRoot` and `--quayStorage` flags. For example:
 +
 [source,terminal]
 ----
-$ sudo ./mirror-registry upgrade --quayHostname <host_example_com> --quayRoot <example_directory_name> --pgStorage <example_directory_name>/pg-data --quayStorage <example_directory_name>/quay-storage -v
+$ sudo ./mirror-registry upgrade --quayHostname <host_example_com> --quayRoot <example_directory_name>  --quayStorage <example_directory_name>/quay-storage -v
```

Comment:
I just saw all of the `sudo` commands and was wondering if you need root privileges.

Reply:
mirror-registry can be running with rootless. `sudo` is not mandatory